internal/cli/down.go

```diff
@@ -64,10 +66,14 @@ func (c *downCmd) run(ctx context.Context, args []string) error {
 			break
 		}
 	}
-	dcc := downDeps.dcClient
-	err = dcc.Down(ctx, dcConfigPaths, logger.Get(ctx).Writer(logger.InfoLvl), logger.Get(ctx).Writer(logger.InfoLvl))
-	if err != nil {
-		logger.Get(ctx).Infof("error running `docker-compose down`: %v", err)
+
+	if len(dcConfigPaths) > 0 {
```

Comment:
ha i was gonna leave this for maria to fix but okay!

Reply:
i couldn't fix the bug without also fixing this problem

Reply:
(or the integration tests would have failed)

Reply:
i figured as much :)